languages/python/sqlalchemy-oso/tests/test_sqlalchemy.py

```diff
@@ -303,3 +309,45 @@ def test_unconditional_policy_has_no_filter(engine, oso, fixture_data):
         + "posts.access_level AS posts_access_level, posts.created_by_id AS posts_created_by_id, "
         + "posts.needs_moderation AS posts_needs_moderation \nFROM posts \nWHERE 1 = 1"
     )
+
+
+def test_authorized_sessionmaker_multiple_actions(engine, oso):
```

Comment:
Added that functionality to the existing test for the relationships 'bug' (the one that was previously xfail-ed) and removed this test since it uses bespoke fixture data and is now duplicative of the other test.